webpack/stories/docs/plugins.md

```diff
@@ -41,7 +41,18 @@ Until this is fixed in Foreman core the workaround is using [mocks](https://gith
 
 ### Using components outside of webpack
 
-Using components from core outside of webpack processed code it's currently **not possible**. In future the `componentRegistry` should be available in `Window.tfm` object, which will unlock the path and enable for example using React components in AngularJS in Katello.
+The component registry is available in `Window.tfm.componentRegistry`. That gives you access to the components even from js code that isn't processed by webpack.
```

Comment:
I just wonder why that would be needed. I can only think of katello using certain components in Angular pages, is that it?

Reply:
Exactly! :) 
(https://github.com/Katello/katello/pull/7001/files#diff-5bcfdae0ace18ad3702fbb46115e768d)